guide/english/php/functions/time/index.md

```diff
@@ -16,3 +16,21 @@ echo time();
 ```
 #### More Information:
 * <a href="https://secure.php.net/manual/en/function.time.php">php.net time() manual</a>
+
+## Default timezone
+
+You can set or get the settings using `date_default_timezone_set()` and `date_default_timezone_get()`.
+
+### Example
+```php
+<?php
+date_default_timezone_set(‘Asia/Dhaka’);  //To set the time zone.
+$variable = date_default_timezone_get();  //To get the timezone.
+```
```

Comment:
Here you haven't echoed the `$variable` so technically there won't be any output, so it is suggested to add an echo keyword such as
`echo $variable`